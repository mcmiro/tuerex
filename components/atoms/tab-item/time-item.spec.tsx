import * as React from "react";
import {screen, render} from "@testing-library/react";
import TimeItem from "./";

describe("Test TimeItem component", () => {
	it("Should pass when TimeItem is defined", () => {
		render(<TimeItem name="test" data-testid="switch-test" />);

		const timeItem = screen.queryAllByTestId("switch-test");
		expect(timeItem).toBeDefined();
		expect(timeItem).toMatchSnapshot();
	});
});
