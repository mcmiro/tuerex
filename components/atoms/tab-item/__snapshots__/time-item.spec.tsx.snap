// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test TimeItem component Should pass when TimeItem is defined 1`] = `[]`;
